composer.json

```diff
@@ -62,7 +62,10 @@
     "autoload-dev": {
         "psr-0": {
             "": "engine/tests/phpunit/"
-        }
+        },
+		"files": [
+			"engine/tests/phpunit/bootstrap.php"
```

Comment:
To clarify, you are saying remove composer autoload, require from TestCase? Should I keep the constant in phpunit?